dotnet/src/Planners/Planners.Core/Handlebars/Models/HandlebarsParameterView.cs

```diff
@@ -0,0 +1,98 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.Generic;
+using System.Text.Json.Serialization;
+
+namespace Microsoft.SemanticKernel.Planners.Handlebars.Models;
+
+internal class HandlebarsParameterTypeView
```

Comment:
We are changing our naming from `View` to `Metadata`, see https://github.com/microsoft/semantic-kernel/pull/3545

Reply:
Refactored everything from `View` -> `Metadata`